src/ui/opengl/castlecontrols.pas

```diff
@@ -104,7 +104,7 @@ implementation
 
 uses SysUtils, Math, CastleControlsImages, CastleTextureFont_DjvSans_20,
   CastleTextureFont_DejaVuSans_10,
-  CastleApplicationProperties;
+  CastleApplicationProperties{$ifdef ANDROID}, CastleMessaging{$endif};
 
```

Comment:
Do not surround `CastleMessaging` use with `{$ifdef ANDROID}`. Use it on all platforms. It will be ignored on platforms that don't support it, which is totally fine. This way adding iOS implementation for this will be trivial -- it will just handle the same message.